synapse/util/retryutils.py

```diff
@@ -97,7 +97,12 @@ def get_retry_limiter(destination, clock, store, ignore_backoff=False, **kwargs)
 
     defer.returnValue(
         RetryDestinationLimiter(
-            destination, clock, store, retry_interval, backoff_on_failure, **kwargs
+            destination,
+            clock,
+            store,
+            retry_interval,
```

Comment:
(I'd be tempted to make them all keyword arguments, but meh)

Reply:
wouldn't have been a bad idea, indeed....